### DISCUSSION TITLE
Installation of compiled kernel and modules on Jetson

### DISCUSSION THREAD

**Discussion by htex0189** [OE4T/meta-tegra#1159] [General]
Hi,
Are there any documents detailing the procedure to install the compiled kernel and modules using the procedure in this wiki https://github.com/OE4T/linux-tegra-5.10/wiki/Building-Linux-tegra-5.10.x-from-sources 
on to Jetson PC?

**Comment by dwalkes** [comment]
I don't know of one specifically for this wiki page but https://github.com/OE4T/meta-tegra/discussions/888 and https://github.com/OE4T/meta-tegra/discussions/602 may be relevant.
